instrumentation/net/http/otelhttp/internal/semconvutil/httpconv.go

```diff
@@ -394,14 +440,22 @@ func (c *httpConv) ServerRequest(server string, req *http.Request) []attribute.K
 // "http.flavor", "net.host.name". The following attributes are
 // returned if they related values are defined in req: "net.host.port".
 func (c *httpConv) ServerRequestMetrics(server string, req *http.Request) []attribute.KeyValue {
-	// TODO: This currently does not add the specification required
-	// `http.target` attribute. It has too high of a cardinality to safely be
-	// added. An alternate should be added, or this comment removed, when it is
-	// addressed by the specification. If it is ultimately decided to continue
-	// not including the attribute, the HTTPTargetKey field of the httpConv
-	// should be removed as well.
-
-	n := 4 // Method, scheme, proto, and host name.
+	/* The following semantic conventions are returned if present:
+	http.scheme				string	Required
+	http.route				string	Conditionally Required: If and only if it’s available
+	http.method				string	Required - Note: Only known good headers and _OTHER
+	http.status_code		int		Conditionally Required: If and only if one was received/sent.
+	net.host.name			string	Required
+	net.host.port			int		Conditionally Required: [4]
+	net.protocol.name		string	Recommended
```

Comment:
This should rather be removed. See

> Recommended: if not default (http)


I am aware that it was not in "metrics" docs but it was rather a bug and this issue is no more in https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md

Reply:
This can be something else. The default server will always be HTTP, but it can respond to spdy or other protocols. So it does set it if it's different from the default.